dahu/core/app/scripts/templates/views/workspace/actions/disappear.html

```diff
@@ -1,6 +1,2 @@
 <div class="disappear">
```

Comment:
Since we are now going to use LayoutView we will not need it at all. 
If and when we will introduce specific way of disappearing we will add it back.
http://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it